docs/Configuring-Pantheon/Networking/Managing-Peers.md

```diff
@@ -0,0 +1,66 @@
+description: Managing Pantheon peers 
+<!--- END of page meta data -->
+
+# Managing Peers 
+ 
+## Limiting Peers
+
+Limiting peers reduces the bandwidth used by Pantheon. It also reduces the CPU time and disk access 
+used to manage and respond to peers.  
+ 
+Use the [`--max-peers`](../../Reference/Pantheon-CLI-Syntax.md#max-peers) command line option to reduce 
+the maximum number of peers. The default is 25.
+
+## No Discovery
+
+The [`--discovery-enabled`](../../Reference/Pantheon-CLI-Syntax.md#discovery-enabled) command line option 
+can be used to disable P2P peer discovery.
+Set this option to `false` if you are running a test node or a network with fixed nodes.
+
+Use the [`admin_addPeer`](../../Reference/JSON-RPC-API-Methods.md#admin_addpeer) JSON-RPC method to add peers
+or configure [static nodes](#static-nodes) when [`--discovery-enabled`](../../Reference/Pantheon-CLI-Syntax.md#discovery-enabled) 
+is set to false. 
+
+## Static Nodes
+
+Use a `static-nodes.json` file to configure static nodes. The node connects to the listed nodes on startup.
+If a connection to a static node goes down, the node attempts to reconnect every 60 seconds. 
+
+The `static-nodes.json` file must be located in the data directory (specified by [`--data-path`](../../Reference/Pantheon-CLI-Syntax.md#data-path))
+and contain a JSON array of [enode URLs](../Node-Keys.md#enode-url).
+
+!!! example 
+    ```json
+    [
+    "enode://cea71cb65a471037e01508cebcc178f176f9d5267bf29507ea1f6431eb6a5dc67d086dc8dc54358a72299dab1161febc5d7af49d1609c69b42b5e54544145d4f@127.0.0.1:30303",
+    "enode://ca05e940488614402705a6b6836288ea902169ecc67a89e1bd5ef94bc0d1933f20be16bc881ffb4be59f521afa8718fc26eec2b0e90f2cd0f44f99bc8103e60f@127.0.0.1:30304"    
+    ]
+    ``` 
+
+!!! note
+    Each node has a `static-nodes.json` file. Ensure each node in the network has the same `static-nodes.json` file. 
+
+Nodes added by [`admin_addPeer`](../../Reference/JSON-RPC-API-Methods.md#admin_addpeer) are not added to the 
```

Comment:
Is it worth saying - if the NodeID in the enode (passed to AdminPeer, or staticNodes) does not match the ACTUAL node ID of the remote peer - the connection will fail (appear to timeout) - but remote node will show a failure regarding "Invalid MAC".
A better error log may be forthcoming, but maybe not soon.